ComfortZoneApp/app/src/main/java/com/example/comfortzone/MainActivity.java

```diff
@@ -11,6 +11,11 @@
 
 import com.google.android.material.bottomnavigation.BottomNavigationView;
 import com.google.android.material.navigation.NavigationBarView;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.widget.Toast;
+
+import com.parse.ParseUser;
 
 public class MainActivity extends AppCompatActivity {
```

Comment:
I think MainActivity is too generic, plus it's not the Main Launcher, we can named it as HostActivity?